iset.mm

```diff
@@ -55980,29 +55999,29 @@ associative structure (such as a group).  (Contributed by NM,
     $( Existence of an operation class abstraction (special case).
        (Contributed by FL, 17-May-2010.)  (Revised by Mario Carneiro,
        1-Sep-2015.) $)
-    mpt2exg $p |- ( ( A e. R /\ B e. S ) -> F e. _V ) $=
-      ( wcel cvv wral elex ralrimivw syl mpt2exxg sylan2 ) DGJZCFJDKJZACLZHKJRS
-      TDGMSSACDKMNOABCDEFKHIPQ $.
+    mpoexg $p |- ( ( A e. R /\ B e. S ) -> F e. _V ) $=
+      ( wcel cvv wral elex ralrimivw syl mpoexxg sylan2 ) DGJZCFJDKJZACLZHKJRST
+      DGMSSACDKMNOABCDEFKHIPQ $.
   $}
 
   ${
     $d x y A $.  $d x y B $.
     $( If the domain of a function given by maps-to notation is a set, the
        function is a set.  (Contributed by NM, 12-Sep-2011.) $)
```

Comment:
"domain of a function" => "domain of an operation"
"the function is a set" => "the operation is a set" 

also for the next theorem